modules/local/linked_adapters.nf

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nextflow
+
+process LINKED_ADAPERS {
```

Comment:
This process doesn't have any output and isn't being used. Maybe just remove this. The code is already included as a separate script in bin/make_linked_adapter.py anyway.